twitter_api/Twitter.py

```diff
@@ -0,0 +1,7 @@
+def PostTwitter(message, api_twitter):
+    if len(message) > 140:
+        print('Post has more of 140 chars')
+        return 1
+    api = api_twitter
+    api.PostUpdate(message)
```

Comment:
```py
try:
  api.PostUpdate(message)
except:
  log.exception("Failed to post to Twitter")
```

Just in case an unexpected issue occurs, or there's an encoding problem.